04 - Guides, Workflows, & Courses/for Knowledge Management.md

```diff
@@ -22,7 +22,7 @@ tags:
 - [GTD in 15 minutes – A Pragmatic Guide to Getting Things Done](https://hamberg.no/gtd)
 
 ### Linking your Thinking
-- [The PARA Method: A Universal System for Organizing Digital Information - Forte Labs](https://fortelabs.co/blog/para/)
+- [Linking Your Thinking - YouTube Channel](https://www.youtube.com/channel/UC85D7ERwhke7wVqskV_DZUA)
```

Comment:
Wait... these two are not the same?

Reply:
yeah, the link to PARA is 3 lines further down. The earlier version simply had the link PARA link two times and hasn't actually linked LYT, which is what I fixed.